http/src/main/resources/reference.conf

```diff
@@ -513,7 +513,9 @@ gke {
     # Templated by firecloud-develop
     postgres.password = "replace-me"
     orchUrl = "https://firecloud-orchestration.dsde-dev.broadinstitute.org/api/"
-    drsUrl = "https://us-central1-broad-dsde-dev.cloudfunctions.net/martha_v3"
+    drsResolver = "martha"
+    marthaUrl = "https://us-central1-broad-dsde-dev.cloudfunctions.net/martha_v3"
+    drshubUrl = "https://drshub.dsde-dev.broadinstitute.org/api/"
```

Comment:
This needs to be `https://drshub.dsde-dev.broadinstitute.org/api/v4/drs/resolve`